west.yml

```diff
@@ -24,10 +24,13 @@ manifest:
     - name: oleksiimoisieiev
       url-base: https://github.com/oleksiimoisieiev
 
+    - name: dsemenets
+      url-base: https://github.com/dsemenets
+
   projects:
     - name: zephyr
-      remote: xen-troops
-      revision: aos-wip
+      remote: dsemenets
+      revision: spider_board
```

Comment:
(question) maybe is it better to wait until I merge related changes to aos-wip?

Reply:
I know. But we have no time ((